### DISCUSSION TITLE
taro1.x升级到3.x命令行工具

### DISCUSSION THREAD

**Discussion by maoliang2271** [NervJS/taro#15325] [生态]
由于Taro1.x官方已不再进行维护，在开发体验上跟3.x版本存在较大的不足，我开发了一个cli工具可以帮忙开发者对已有Taro1.x项目一键升级到Taro3.x版本，欢迎大家使用、共建及提出改进意见🤝🤝，详见 https://github.com/maoliang2271/TaroVersionUpTools

**Comment by eatureide** [comment]
2呢？
